api/reference/1.0/notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+---
+
+# Notes
+
+`Notes` are textual annotations you can add to Launch resources of these types:
+- `DataElements`
+- `RuleComponents`
+- `Rules`
+- `Extensions`
+- `Libraries`
+- `Properties`
```

Comment:
Alphabetize these?

Reply:
I was sorting them from simplest to most complex, but... okay.